lib/tasks/cortex/core/media.rake

```diff
@@ -164,7 +164,7 @@ namespace :cortex do
                 "name": "Creator",
                 "cells": [{
                             "field": {
-                              "method": "author_email"
+                              "method": "creator.email"
```

Comment:
this might make it more difficult to translate to React although it technically could work, just in a less elegant way.

Reply:
This has the benefit of cleaning up Cortex's domain - we won't have a ton of useless methods directly on a Model. When we approach the React refactor, we could look at bringing similar functionality back by using a Plugin Transaction.

I'm not following how this will make it more difficult with React, however - both `author_email` and `creator.email` are going to be difficult for React to work with, as they both require separate queries to the API for their information (unless we batch these requests during prerender). We'll have to discuss how we approach this.

Reply:
> they both require separate queries to the API for their information (unless we batch these requests during prerender). We'll have to discuss how we approach this.

Yeah, on second thought it's too early to worry about field rendering methods at the moment. But we should definitely look at improving our db indexing so that we only need to hit the database once.